ledspicer/data/animations/Pulse.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<LEDSpicer version="1.0" type="Animation">
+    <actor
+        type="PipeWire"
+        group="Player1"
+        filter="Normal"
+        mode="VuMeter"
+        direction="Forward"
+        off="DarkCyan"
+        low="Blue"
+        mid="Red"
+        high="White"
+        channel="Both"
+    />
+    <actor
+        type="PipeWire"
```

Comment:
The plugin PipeWire is not ready yet, it works but I was unable to get any output. I suggest using PulseAudio plugin instead.